elastic/kibana
# [ResponseOps] ES|QL rule type improvements - write query results to the alert doc

Resolves https://github.com/elastic/response-ops-team/issues/200

## Summary

This PR copies the fields from the ES|QL query results to the alert doc. Now that we are writing everything to the alert doc I removed the source fields selector from the ES|QL ui.


### Checklist



### To verify

- Create an ES Query ES|QL rule.
- Go to [dev tools](http://localhost:5601/app/dev_tools#/console) and run the query below to verify that the query results are written to the alert doc.
```
GET .internal.alerts-stack.alerts-default*/_search
```
